man/docker-run.1.md

```diff
@@ -508,6 +509,18 @@ content label. Shared volume labels allow all containers to read/write content.
 The `Z` option tells Docker to label the content with a private unshared label.
 Only the current container can use a private volume.
 
+To control mount propagation property of volume one can use `:[r]shared`,
+`:[r]slave` or `:[r]private` propagation flag. Propagation property can
+be specified only for bind mounted volumes and not for internal volumes or
+volume driver volumes. For this to work source mount point (mount point where
+source dir is mounted on) has to have right propagation properties. For shared
+volumes, source mount point has to be shared. And for slave volumes, source
+mount has to be either shared or slave. Use `df <source-dir>` to figure out
+the source mount and then use `findmnt -o TARGET,PROPAGATION <source-mount-dir>`
+ to figure out propagation properties of source mount. To change propagation
+properties of a mount point use `mount` command. For example,
+`mount --make-shared /`.
```

Comment:
> Having said that we run with `/` being shared in `fedora` and things seem to be fine.

Yup, I understood that. But I wasn't sure if it would make a difference on other distros; people *will* copy/paste examples and we should prevent bringing them in trouble if not necessary.